sdks/java/core/src/main/java/org/apache/beam/sdk/testing/TestPipeline.java

```diff
@@ -58,23 +57,21 @@
  * A creator of test pipelines that can be used inside of tests that can be configured to run
  * locally or against a remote pipeline runner.
  *
- * <p>It is recommended to tag hand-selected tests for this purpose using the {@link
- * ValidatesRunner} {@link Category} annotation, as each test run against a pipeline runner will
- * utilize resources of that pipeline runner.
+ * <p>By default, the pipeline will use options from {@link PipelineOptionsFactory#create} and use
+ * the direct runner. Options can be configured using the system property "beamTestPipelineOptions":
```

Comment:
`the {@code DirectRunner}`?

Specifically it will do so if the `DirectRunner` is on the classpath. For users who do not normally have the `DirectRunner` on the classpath, they should consider adding it in the `test` scope.